client/src/components/CatalogMainSection/CatalogMainSection.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Stack, Typography, Container } from '@mui/material';
+import styles from './CatalogMainSection.module.scss';
+
+function CatalogMainSection() {
+  return (
+    <Stack className={styles.mainSection} justifyContent="center">
+      <Container>
+        <Typography
+          variant="h1"
+          color="#FFFFFF"
```

Comment:
please use our theme for the color value since we have it in our default theme and it is accessible within mui components.

Reply:
done